aws/v20.1.5/release.diff

```diff
@@ -0,0 +1,187 @@
+    name: aws-ebs-csi-driver                                         name: aws-ebs-csi-driver
+    version: 2.30.1                                                  version: 2.30.1
+  - dependsOn:                                                     - dependsOn:
+    - aws-ebs-csi-driver                                             - aws-ebs-csi-driver
+    name: aws-ebs-csi-driver-servicemonitors                         name: aws-ebs-csi-driver-servicemonitors
+    version: 0.1.0                                                   version: 0.1.0
+  - dependsOn:                                                     - dependsOn:
+    - aws-cloud-controller-manager                                   - aws-cloud-controller-manager
+    - cilium                                                         - cilium
+    - coredns                                                        - coredns
+    - prometheus-operator-crd                                        - prometheus-operator-crd
+    - vertical-pod-autoscaler-crd                                    - vertical-pod-autoscaler-crd
+    name: cert-exporter                                              name: cert-exporter
+    version: 2.9.0                                                   version: 2.9.0
+  - dependsOn:                                                     - dependsOn:
+    - aws-cloud-controller-manager                                   - aws-cloud-controller-manager
+    - cilium                                                         - cilium
+    - coredns                                                        - coredns
+    - prometheus-operator-crd                                        - prometheus-operator-crd
+    - vertical-pod-autoscaler-crd                                    - vertical-pod-autoscaler-crd
+    name: cert-manager                                               name: cert-manager
+    version: 3.7.9                                                   version: 3.7.9
+  - name: chart-operator                                           - name: chart-operator
+    version: 3.2.1                                                   version: 3.2.1
+  - dependsOn:                                                     - dependsOn:
+    - prometheus-operator-crd                                        - prometheus-operator-crd
+    name: chart-operator-extensions                                  name: chart-operator-extensions
+    version: 1.1.2                                                   version: 1.1.2
+  - componentVersion: 1.15.1                                   |   - dependsOn:
+    dependsOn:                                                 < 
+    - aws-cloud-controller-manager                                   - aws-cloud-controller-manager
+    name: cilium                                                     name: cilium
+    version: 0.22.0                                            |     version: 0.25.1
```

Comment:
Please also re-generate the diff with the component version re-added to the `release.yaml`.

Reply:
IIRC you can pass the component version to the `devctl` command.